iml-graphql-queries/src/filesystem.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+pub mod ldev_create {
```

Comment:
I could see this being under snapshots as well.

Reply:
Should I move it to snapshot.rs?